packages/studio/styled-system/reset.css

```diff
@@ -1,4 +1,4 @@
-@layer reset {
+@layer @layer reset {} {
```

Comment:
caught this, maybe the studio hasnt been updated and you fixed it since, not sure